src/main/java/us/icarenow/web/service/UserService.java

```diff
@@ -0,0 +1,22 @@
+package us.icarenow.web.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import us.icarenow.web.controller.form.SignUpPatientForm;
+import us.icarenow.web.model.entity.Role;
+import us.icarenow.web.model.entity.User;
+import us.icarenow.web.repository.UserRepository;
+
+@Service
+public class UserService {
```

Comment:
Then please wait for Adi's changes. Once his changes are in main branch then update your branchand adapt to his changes